src/sparseml/exporters/transforms/delete_repeated_qdq.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from onnx import ModelProto
+
+from sparseml.exporters.transforms.onnx_transform import OnnxTransform
+from sparseml.onnx.utils.graph_editor import ONNXGraph
+
+
+__all__ = ["DeleteRepeatedQdq"]
+
+
+class DeleteRepeatedQdq(OnnxTransform):
+    """
+    Transforms
+    ```
+    QuantizeLinear
+        |
+    DequantizeLinear
+        |
+    QuantizeLinear
+        |
+    DequantizeLinear
+    ```
+    Into
+    ```
+    QuantizeLinear
+        |
+    DequantizeLinear
+    ```
+    """
+
+    def transform(self, model: ModelProto) -> ModelProto:
+        graph = ONNXGraph(model)
+        nodes_to_delete = []
+        quant_nodes = [n for n in model.graph.node if n.op_type == "QuantizeLinear"]
+        for quant_node_1 in quant_nodes:
+            dequant_node_1 = graph.get_node_single_child(quant_node_1)
+            if not dequant_node_1 or dequant_node_1.op_type != "DequantizeLinear":
```

Comment:
We have a helper function for that:
```suggestion
            if not assert_op_type(dequant_node_1, "DequantizeLinear"):
```